src/pages/civic/index.vue

```diff
@@ -147,8 +142,12 @@
                 .flex.justify-center.mb-24(
                   v-if="isExperimenting"
                 )
-                  LikeButtonAnimation(:size="180")
+                  LikeButtonAnimation(
+                    :size="180"
+                    :avatar="referrer.avatar"
```

Comment:
will this explode if referrer is undefined?

Reply:
I have guarded in L243

Reply:
After we end the experiment, L243 would be removed, and this part of template might be applied.
Should also guard in template?